### DISCUSSION TITLE
Installing a Chinese font in u8g2

### DISCUSSION THREAD

**Discussion by Gudzys** [olikraus/u8g2#2056] [Q&A]
Am trying to run Shennong sketch and get this error: 

/home/brent/Arduino/libraries/U8g2/src/U8g2lib.h:266: undefined reference to `u8g2_font_wqy14_t_gb2312b'
/usr/bin/avr-ld: /home/brent/Arduino/libraries/U8g2/src/U8g2lib.h:266: undefined reference to `u8g2_font_wqy14_t_gb2312b'
collect2: error: ld returned 1 exit status
Using library U8g2 at version 2.33.15 in folder: /home/brent/Arduino/libraries/U8g2 
Using library SPI at version 1.0 in folder: /usr/share/arduino/hardware/archlinux-arduino/avr/libraries/SPI 
Using library Wire at version 1.0 in folder: /usr/share/arduino/hardware/archlinux-arduino/avr/libraries/Wire 
exit status 1
Error compiling for board Arduino Mega or Mega 2560.

Have the fonts installed at /home/brent/Arduino/libraries/u8g2_wqy-master Where should it be installed for Arduino to see it?

Thanks.

**Comment by olikraus** [comment]
The font is to large for your target: `U8G2_USE_LARGE_FONTS` is not defined.
You probably need to use a different font instead.

You could also add the line
```
#define U8G2_USE_LARGE_FONTS
```
At the beginning of u8g2.h but you may run into other problems.

**Comment by Gudzys** [comment]
Xi xi.

Sent with [Proton Mail](https://proton.me/) secure email.

------- Original Message -------
